source/common/http/conn_manager_utility.cc

```diff
@@ -85,6 +85,9 @@ Network::Address::InstanceConstSharedPtr ConnectionManagerUtility::mutateRequest
   Network::Address::InstanceConstSharedPtr final_remote_address;
   bool single_xff_address;
   const uint32_t xff_num_trusted_hops = config.xffNumTrustedHops();
+  bool trusted_forwarded_proto =
+      Runtime::runtimeFeatureEnabled("envoy.reloadable_features.trusted_forwarded_proto");
```

Comment:
Why not make this true by default? I guess do we have a documented procedure of the sequence for these type of features? false by default -> true by default -> remove old code path?

Reply:
Yeah, generally false -> true -> remove, but for bugfixes and low risk code I think it's fine to set true by default so done.

Reply:
Not related to this PR, but do your scripts track this somehow? I think this is all awesome, I just want to make sure that we don't forget about the sequence of things we need to do upon each release?

Reply:
Yep, the scripts grep through the whole code base and add any flags which aren't already defaulted true.